ckanext/datagovtheme/fanstatic_library/scripts/geoplatform.js

```diff
@@ -23,4 +23,6 @@ async function getGeoplatformLink() {
     }
 }
 
-getGeoplatformLink();
+jQuery(function ($) {
```

Comment:
did this not work?

Reply:
i would have thought that we'd not need to wrap the entire function.

Reply:
it did work but @FuhuXia recommended wrapping the entire function to avoid adding the function to the window scope

Reply:
Fair enough. That makes sense. 👍🏼